docs/source-pytorch/model/manual_optimization.rst

```diff
@@ -186,8 +186,8 @@ Use ``self.lr_schedulers()`` in  your :class:`~pytorch_lightning.core.module.Lig
 defined in your :meth:`~pytorch_lightning.core.module.LightningModule.configure_optimizers`.
 
 .. warning::
-   * Before v1.3, Lightning automatically called ``lr_scheduler.step()`` in both automatic and manual optimization. From
-     1.3, ``lr_scheduler.step()`` is now for the user to call at arbitrary intervals.
+   * Before v1.3, Lightning automatically called ``lr_scheduler.step()`` in both automatic and manual optimization. 1.3, ``lr_scheduler.step()`` can now be called at arbitrary intervals by the user, or automatically by Lightning if ``"interval"`` is defined in ``configure_optimizers()``.
+     Check ``configure_optimizers()`` for more details.
```

Comment:
```suggestion
   * Before v1.3, Lightning automatically called ``lr_scheduler.step()`` in both automatic and manual optimization. Post v1.3, ``lr_scheduler.step()`` can now be called at arbitrary intervals by the user in case of manual optimization, or by Lightning if ``"interval"`` is defined in :meth:`~pytorch_lightning.core.module.LightningModule.configure_optimizers` in case of automatic optimization.
```
I think this is better.